webui/src/lib/components/repository/changes.jsx

```diff
@@ -52,6 +54,8 @@ const ChangeRowActions = ({ entry, onRevert }) => {
     entry: The entry the TreeItem is representing, could be either an object or a prefix.
     repo: Repository
     reference: commitID / branch
+    compareReference: commitID / branch
+    displayedAt: the page in which the TreeItem is displayed. e.g. changes, commit, or compare pages.
```

Comment:
order it based on the function arguments order

Reply:
display at - problematic property. it bound the view to the application business. is there a way to transform it to a visual property/properties we can pass instead of saying the place we render it.